x-pack/plugins/cases/public/components/case_action_bar/index.test.tsx

```diff
@@ -249,4 +223,38 @@ describe('CaseActionBar', () => {
     userEvent.click(screen.getByTestId('property-actions-ellipses'));
     expect(queryByText('Delete case')).toBeInTheDocument();
   });
+
+  it('shows the external incident action', async () => {
+    const connector = caseConnectors['servicenow-1'];
+    const { push, ...connectorWithoutPush } = connector;
```

Comment:
If this is a `connectorWithoutPush` should the popout be there?

Reply:
The component to get the information from the `useGetCaseConnectors ` does `caseConnectors?.[caseData.connector.id]?.push.externalService ?? null;`. I need to have a valid connector ID in the case data so this `caseConnectors[caseData.connector.id]` will work.